typescript-functions/main.ts

```diff
@@ -0,0 +1,53 @@
+interface Person {
+  firstName: string;
+  lastName: string;
+}
+
+function convertMinutesToSeconds(minutes: number): number {
+  return minutes * 60;
+}
+
+console.log(
+  'Convert minutes to Seconds for a 10 minute argument',
+  convertMinutesToSeconds(10)
+);
+
+const greet = (name: string): string => {
+  return `Hey ${name}!`;
+};
```

Comment:
Arrow functions also have a different type of syntax referred to as "concise" body syntax, which will allow you to forgo the `return` keyword
```suggestion
const greet = (name: string): string => `Hey ${name}!`;
```